airbyte-integrations/connectors/source-delighted/source_delighted/source.py

```diff
@@ -157,7 +165,7 @@ def check_connection(self, logger, config) -> Tuple[bool, any]:
 
         try:
             auth = self._get_authenticator(config)
-            args = {"authenticator": auth, "since": config["since"]}
+            args = {"authenticator": auth, "since": self.toUnix(config)}
```

Comment:
```suggestion
            args = {"authenticator": auth, "since": pendulum.parse(config["since"])}
```

I would suggest to make the `since` attribute a datetime and to leverage pendulum for easy iso date parsing. 
You can then create a `since_ts` property on the stream like: 

```python
@property 
def since_ts(self) -> int:
  return int(self.since.timestamp())
```

And finally use `self.since_ts` in `requests_params`.

I think this will add a lot more flexibility and clarity in term of date manipulation. What do you think?